modules/andes-core/broker/src/main/java/org/wso2/andes/store/FailureObservingAndesContextStore.java

```diff
@@ -418,4 +422,359 @@ private synchronized void notifyFailures(AndesStoreUnavailableException e) {
         }
     }
 
+
+    /**
+     * Create a new slot in store
+     *
+     * @param startMessageId   start message id of slot
+     * @param endMessageId     end message id of slot
+     * @param storageQueueName name of storage queue name
+     * @param assignedNodeId
+     * @throws AndesException
+     */
+    @Override
+    public void createSlot(long startMessageId, long endMessageId, String storageQueueName, String assignedNodeId) throws AndesException {
+        try {
+            wrappedInstance.createSlot(startMessageId, endMessageId, storageQueueName, assignedNodeId);
```

Comment:
Since "wrappedInstance" refers to the AndesContextStore, can use a better, relevant name for clarity.